apps/cli/src/commands/create.ts

```diff
@@ -36,28 +35,52 @@ export default class CreateCommand extends Command {
         }),
         branch: Flags.string({
             description: `sunodo/sunodo-templates repository branch name to use`,
-            default: DEFAULT_TEMPLATES_BRANCH,
+            default: "",
+        }),
+        repository: Flags.string({
+            description: `Define a different repository to use. Supports GitHub, GitLab, bitbucket and sourcehut. Example: --repository https://github.com/sunodo/sunodo-templates`,
         }),
     };
 
     private async download(
         template: string,
         branch: string,
         out: string,
+        repository?: string,
     ): Promise<DownloadTemplateResult> {
-        const sunodoProvider: TemplateProvider = async (input) => {
-            return {
-                name: "sunodo",
-                subdir: input,
-                url: "https://github.com/sunodo/sunodo-templates",
-                tar: `https://codeload.github.com/sunodo/sunodo-templates/tar.gz/refs/heads/${branch}`,
-            };
-        };
+        let source;
+
+        if (repository) {
+            if (repository.includes("github.com")) {
```

Comment:
I think that if we will continue to use the [gitget](https://www.npmjs.com/package/giget#examples) package, which is a little bloated, we should let it do the work, and don't try to do this extra processing here, because it adds maintenance.

Reply:
There's a "crazy" approach I used at #345, and we get rid of giget in the process.

Reply:
That just increase our [dependency on Docker](https://github.com/sunodo/sunodo/issues/367)

Reply:
That's the reason for the quotes on "crazy". :)

Reply:
> I think that if we will continue to use the [gitget](https://www.npmjs.com/package/giget#examples) package, which is a little bloated, we should let it do the work, and don't try to do this extra processing here, because it adds maintenance.

Need to convert repository URLs to "github:owner/repo" or "gitlab:owner/repo" and ... format for the `downloadTemplate` function. Currently, directly using `downloadTemplate`, requiring manual URL conversion.

Example:

```code
const { source, dir } = await downloadTemplate("github:unjs/template");
```

Reply:
We have two options: 

1 - get the full repo address from user and then convert it to the appropriate format (Like what I did)
2 - add a flag to define the SCM type. (Like: `--repo-type` )
       example:
       
       ```shell
       sunodo create test --template python --repository omidasadpour/sunodo-templates  --branch main --repo-type github
       ```

Reply:
I mean documenting the format that giget is expecting (i.e. gh:org/repo).
In any case the default should still be download from sunodo templates.